addons/mass_mailing/static/src/js/wysiwyg.js

```diff
@@ -27,7 +27,7 @@ const MassMailingWysiwyg = Wysiwyg.extend({
         const initialDropZone = this.$editable[0].querySelector('.o_mail_wrapper_td');
         const parsedHtml = new DOMParser().parseFromString(currentValue, "text/html");
         if (initialDropZone && !parsedHtml.querySelector('.o_mail_wrapper_td')) {
-            initialDropZone.replaceChildren(currentValue);
+            initialDropZone.innerHTML = currentValue;
```

Comment:
Wouldn't this work too ? (and would not trigger the security ci)
```js
const parsedHtml = parseHtml(currentValue);
if (initialDropZone && !parsedHtml.querySelector('.o_mail_wrapper_td')) {. 
    initialDropZone.replaceChildren(parsedHtml);
```

Reply:
@dmo-odoo Are you talking about `parseHTML` function which we have in 16.0 ? I don't think we have anything like that in 15.0?

Reply:
I meant that function yes but you're right that it isn't in 15.0 yet. However, I think you can used the `parsedHtml` variable anyway. Something, maybe not exactly that, but conceptually like this:
```js
initialDropzone.replaceChildre(...parsedHtml.childNodes);
```

Reply:
Done changes. Please taske a look.,